### DISCUSSION TITLE
Is the SOH of the battery available for input?

### DISCUSSION THREAD

**Discussion by RobinBluehub** [pybamm-team/PyBaMM#2595] [Q&A] [answered]
Can pybamm implement aging studies on batteries with different SOH? If it is possible, how to input a different SOH?I sincerely hope the Pybamm team will have the time to respond to my query.

**Comment by brosaplanella** [comment, marked as answer]
Hi! Different people define SoH in different ways, so we don't have it built in PyBaMM, but most definitions can be easily defined with PyBaMM's current functionality. For example, if you check the Summary variables section in the [long experiment notebook](https://github.com/pybamm-team/PyBaMM/blob/develop/examples/notebooks/simulating-long-experiments.ipynb), you can find how to extract the capacity which you can then use to calculate SoH.
